src/OpenLoco/src/GameCommands/Buildings/CreateBuilding.cpp

```diff
@@ -124,7 +124,7 @@ namespace OpenLoco::GameCommands
                             // even if a clear could succeed here. This is
                             // because if it did place a ghost the ghost cleanup
                             // function might remove the wrong building!
-                            if (flags & Flags::flag_1)
+                            if (flags & Flags::skipGhostPlacement)
```

Comment:
its not quite a skipGhostPlacement its more a prevent building clearing. And this is because otherwise you couldn't remove the correct ghost image as the remove functions aren't sophisticated enough.

Reply:
Would `Flags::preventBuildingClearing` be a good new name then?